src/components/NFTMessage/Identity.vue

```diff
@@ -92,6 +96,9 @@ export default {
       });
     },
     userLabel() {
+      if (this.customUserLabel) {
+        return this.customUserLabel;
+      }
```

Comment:
Can we just pass a new type `publisher` and change all `this.type === 'creator'` logic to `creator || publisher` accordingly?

Reply:
[🎨 Use type props to identify user label](https://github.com/likecoin/liker-land/pull/1821/commits/341180c2ae62d87f4b8bc00067a275a98659b4e1)